addons/test_mail/tests/test_message_post.py

```diff
@@ -162,6 +165,29 @@ def test_notify_by_email_add_signature_no_author_user_or_no_user(self):
         template_values = test_record._notify_by_email_prepare_rendering_context(test_message, {})
         self.assertEqual(template_values['signature'], '')
 
+    def test_notify_by_email_prepare_rendering_context_subtitle(self):
```

Comment:
Very nitpicky, but this should be after the following test, to keep the tests in alphabetic order -> in order of execution.